Engine/ac/character.cpp

```diff
@@ -1558,6 +1558,20 @@ void Character_SetTransparency(CharacterInfo *chaa, int trans) {
     chaa->transparency = GfxDef::Trans100ToLegacyTrans255(trans);
 }
 
+int Character_GetBlendMode(CharacterInfo *chaa) {
+
+    return charextra[chaa->index_id].blend_mode;
+}
+
+void Character_SetBlendMode(CharacterInfo *chaa, int blendMode) {
+    /*
+    // TODO validation when blendmodes finalized
+    if ((blendMode < 0) || (blendMode > 4))
+    quit("!SetBlendMode: invalid blend mode");
+    */
```

Comment:
I guess this commented code can be restored now?
Also, what's the meaning of "finalize" here, cannot blend mode be extended in the future by adding more constants to enum, or you think that will break something?